modules/Bio/EnsEMBL/Variation/DBSQL/vcf_config.json

```diff
@@ -135,52 +135,61 @@
       }
     },
     {
-      "id": "gnomADg_GRCh38",
-      "description": "Genome Aggregation Database genomes r2.1",
+      "id": "gnomADg_r3.0_GRCh38",
+      "description": "Genome Aggregation Database genomes r3.0",
       "species": "homo_sapiens",
       "assembly": "GRCh38",
       "type": "remote",
-      "filename_template": "ftp://ftp.ensembl.org/pub/data_files/homo_sapiens/GRCh38/variation_genotype/gnomad/r2.1/genomes/gnomad.genomes.r2.1.sites.grch38.chr###CHR###_noVEP.vcf.gz",
+      "filename_template": "ftp://ftp.ensembl.org/pub/data_files/homo_sapiens/GRCh38/variation_genotype/gnomad/r3.0/genomes/gnomad.genomes.r3.0.sites.chr###CHR###_trimmed_info.vcf.bgz",
```

Comment:
Are these files only made available on release of 100?

Reply:
yes